runtime/runtime.go

```diff
@@ -14,6 +14,9 @@ type Interface interface {
 	Hook(cluster *v1.Cluster) error
 	Upgrade(cluster *v1.Cluster) error
 	Reset(cluster *v1.Cluster) error
+	CNI(cluster *v1.Cluster) error
```

Comment:
delete those interfaces, Apply does not care about those logic.

Reply:
apply cni config, but I don't modify node and master config.